skl2onnx/operator_converters/CalibratedClassifierCV.py

```diff
@@ -46,61 +48,61 @@ def convert_calibrated_classifier_base_estimator(scope, operator, container, mod
     # ARRAYFEATUREEXTRACTOR <- k [1] -> ARRAYFEATUREEXTRACTOR
     #         |                          |
     #         V                          V
-    #  transposed_df_col[M, 1]        transposed_df_col[M, 1]
```

Comment:
I think we should increase the limit on line size. I think the graph below should be part of the docstring and not in comment. I'm ok to break the limit here if it is easier to read.